android/quest/.gitignore

```
@@ -1,4 +1,3 @@
 /build
-
 release/
-
+/src/zeir/assets/configs/
```

we can make this generic

```suggestion
/src/*/assets/configs/
```